### ISSUE TITLE
Incorrect column in parse error

### ISSUE THREAD

**Issue by btmills** [eslint/espree#173]
Originally discovered by @feross in eslint/eslint#3128:

```js
// test.js
try {
    require("espree").parse("function fn(a, a) {\n}", { ecmaFeatures: { modules: true } });
} catch (err) {
    console.error(require("util").inspect(err));
}
```

```sh
$ node test.js
{ [Error: Line 1: Strict mode function may not have duplicate parameter names]
  index: 15,
  lineNumber: 1,
  column: -4,
  description: 'Strict mode function may not have duplicate parameter names' }
```

Notice that the column number is -4, when it should be 16 (if I counted correctly). The `\n` is required, or the column will be correct.

**Comment by nzakas** [comment]
Yeah, that's weird.

**Comment by nzakas** [comment]
Working on this
